browser/ui/views/frame/vertical_tab_strip_widget_delegate_view.cc

```diff
@@ -112,7 +112,9 @@ void VerticalTabStripWidgetDelegateView::UpdateWidgetBounds() {
     return;
 
   auto* widget = GetWidget();
-  DCHECK(widget);
+  if (!widget)
```

Comment:
Just curious what makes this change?

Reply:
Yeah, it starts to fail to pass the DCHECK on the startup.